hypervisor/dm/vpci/core.c

```diff
@@ -30,7 +30,7 @@
 #include <hypervisor.h>
 #include "pci_priv.h"
 
-inline uint32_t pci_vdev_read_cfg(struct pci_vdev *vdev, uint32_t offset, uint32_t bytes)
+uint32_t pci_vdev_read_cfg(struct pci_vdev *vdev, uint32_t offset, uint32_t bytes)
```

Comment:
Here, if we add "static", can it solve the violation?  If YES, I prefer to add static.  Please make sure all internal APIs should use static. Otherwise, we need documentation effort for external APIs

Reply:
This is in a .c file and is called by other files in dm/vpci, so can't add the static modifier.

Reply:
ok